src/main/java/com/wcc/platform/domain/cms/attributes/Experience.java

```diff
@@ -0,0 +1,24 @@
+package com.wcc.platform.domain.cms.attributes;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Experience {
```

Comment:
We discussed with Steph to make this integer and for filters we make the change, so we only return the exactly experience.